### PR TITLE
Running commands locally

### PR DESCRIPTION
**Pull request by yob-yob**
This PR should allow you to run commands locally just line Laravel/envoy buy using `127.0.0.1` / `localhost` / `local` as your `$host`

### CODE REVIEW

**Comment by yob-yob** [comment]
if you have meant for this package to only focus on running commands to a server via ssh and you're not planning to let this package run commands locally because that is not included in the scope of this package then feel free to close this PR.

Thank you.

File: tests/SshTest.php [hunk 1]
```diff
@@ -149,4 +150,13 @@ public function it_can_configure_the_used_process()
...
         $this->assertMatchesSnapshot($command);
     }
+
+    /** @test */
+    public function it_can_run_a_command_locally()
+    {
+        $local = new Ssh('user', '127.0.0.1');
+        $command = $local->execute('whoami');
+
+        Assert::assertEquals(trim($command->getOutput()), get_current_user());
```
Comment (freekmurze) [inline tests/SshTest.php:160]: Use `$this->assertEquals()`

**Review by freekmurze — CHANGES_REQUESTED** [review]
Could you also make sure all tests pass?

File: tests/SshTest.php (hunk 1 above)
Comment (yob-yob) [reply tests/SshTest.php]: Updated this so that the code uses `$this->assertEquals()`

**Comment by yob-yob** [comment]
Sorry if I've handled this late... but Test should now be passing...

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**